cmake/BuildLibCURL.cmake

```diff
@@ -150,8 +150,7 @@ macro(curl_build)
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_RUSTLS=OFF")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_WOLFSSH=OFF")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_GSASL=OFF")
-    list(APPEND LIBCURL_CMAKE_FLAGS "-DHAVE_LIBIDN2=OFF")
```

Comment:
Should we put `DHAVE_LIBIDN2` to `cmake/curl-excluded-options.txt`?

Reply:
I've verified that it is not in the `cmake -LHA` output, at least on curl `8.10.1`. It seems, now there is no point to place it neither to the passed options, nor to the ignored ones.

Reply:
I agree with this changes, `USE_LIBIDN2` was set to ON by default in `curl/CMakeLists.txt` so I didn't know that we don't depend on it.

Reply:
Thanks for explanation!